charts/spire/values.yaml

```diff
@@ -24,6 +24,9 @@ global:
       ## @param global.spire.image.registry Override all Spire image registries at once
       registry: ""
 
+    ## @param global.spire.productionChecks Check values, such as trustDomain, are suitable for production deployment
+    productionChecks: false
```

Comment:
Shall we call this `strictMode`? that is also something used quite commonly in Javascript, React, Typescript world.

Reply:
@faisal-memon ?